src/client/Microsoft.Identity.Client/Extensibility/ExternalBoundTokenScheme.cs

```diff
@@ -11,17 +11,19 @@ namespace Microsoft.Identity.Client.Extensibility
     internal class ExternalBoundTokenScheme : IAuthenticationScheme
     {
         private readonly string _keyId;
+        private readonly string _tokenType;
 
-        public ExternalBoundTokenScheme(string keyId)
+        public ExternalBoundTokenScheme(string keyId, string tokenType = "Bearer")
         {
             _keyId = keyId;
+            _tokenType = tokenType;
         }
 
-        public string AuthorizationHeaderPrefix => "Bearer";
+        public string AuthorizationHeaderPrefix => _tokenType;
```

Comment:
Might have to update the comment [here](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/main/src/client/Microsoft.Identity.Client/AuthenticationResult.cs#L276-L277) as it can be bearer/ pop token